nbconvert/filters/tests/test_markdown.py

```diff
@@ -175,8 +183,8 @@ def test_markdown2rst(self):
         tokens[1] = r'\*\*test'
 
         for index, test in enumerate(self.tests):
-            self._try_markdown(markdown2rst, test, tokens[index])
-
+            self._try_markdown(partial(convert_pandoc, from_format='markdown',
```

Comment:
The second case is only when you are in a template, and you pipe into the filter. 
If you want to be more explicit write the second a case as `... | convert_pandoc('markdown', 'latex') | ... ` than one can see how to use it from within jinja.